pkg/apis/networking/v1beta1/conversion.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	v1beta1 "k8s.io/api/networking/v1beta1"
+	conversion "k8s.io/apimachinery/pkg/conversion"
+	runtime "k8s.io/apimachinery/pkg/runtime"
+	networking "k8s.io/kubernetes/pkg/apis/networking"
+)
+
+func addConversionFuncs(s *runtime.Scheme) error {
+	return s.AddConversionFuncs(
+		Convert_v1beta1_IngressSpec_To_networking_IngressSpec,
+		Convert_networking_IngressSpec_To_v1beta1_IngressSpec,
+	)
+}
+
+func Convert_v1beta1_IngressSpec_To_networking_IngressSpec(in *v1beta1.IngressSpec, out *networking.IngressSpec, s conversion.Scope) error {
```

Comment:
same comment about autoConvert_v1beta1_IngressSpec_To_networking_IngressSpec

Reply:
The original code here probably pre-dates the `autoConvert` trick, but I agree.  Call autoConvert and then fix up the things that it doesn't handle.